Directory.Build.props

```diff
@@ -22,6 +22,11 @@
     <Product>Microsoft ASP.NET Core</Product>
   </PropertyGroup>
 
+  <PropertyGroup Label="Package Information">
+    <Description>A collection of adapters that help with migrating from System.Web.dll based ASP.NET projects to ASP.NET Core projects by providing implementations of common System.Web APIs for use in ASP.NET Core apps.</Description>
```

Comment:
The description should be unique per package.

Reply:
done